govcd/vapp.go

```diff
@@ -384,10 +387,10 @@ func (vapp *VApp) Customize(computername, script string, changeSid bool) (Task,
 		HREF:                vapp.VApp.Children.VM[0].HREF,
 		Type:                types.MimeGuestCustomizationSection,
 		Info:                "Specifies Guest OS Customization Settings",
-		Enabled:             true,
+		Enabled:             takeBoolPointer(true),
```

Comment:
Maybe it is worth it also change this harcoded value too?

Reply:
This function does not have a parameter for this field, while `changeSid` had, but was missed.